h/cli/commands/devserver.py

```diff
@@ -103,7 +103,7 @@ def devserver(https, web, ws, worker, assets, beat):
         m.add_process("beat", "hypothesis --dev celery beat")
 
     if assets:
-        m.add_process("assets", "gulp watch")
+        m.add_process("assets", "node_modules/.bin/gulp watch")
```

Comment:
This was the one place in our code that was trying to use a system-wide copy of gulp, instead of using the one we install in node_modules. Makefile commands already use the node_modules copy for example. Fixed